client/coral-plugin-commentbox/CommentBox.js

```diff
@@ -68,6 +101,15 @@ class CommentBox extends Component {
     this.setState({body: ''});
   }
 
+  addCommentHooks = (hooks = {}) => {
+    if (typeof hooks === 'object') {
```

Comment:
Using objects that gets merged into a single one will result in naming conflicts. Maybe a list would be better?